core-web/libs/contenttype-fields/src/lib/fields/binary-field/binary-field.component.spec.ts

```diff
@@ -351,3 +352,54 @@ describe('DotBinaryFieldComponent', () => {
         jest.resetAllMocks();
     });
 });
+
+@Component({
+    selector: 'dot-app-mock-form',
+    imports: [
+        ButtonModule,
+        DialogModule,
+        MonacoEditorModule,
+        ReactiveFormsModule,
+        DotBinaryFieldComponent
+    ],
+    standalone: true,
+    template: `
+        <form [formGroup]="form">
+            <dot-binary-field [field]="field" formControlName="binaryField"></dot-binary-field>
+        </form>
+    `
+})
+class MockFormComponent {
+    field = FIELD;
+    form = new FormGroup({
+        binaryField: new FormControl('')
+    });
+}
```

Comment:
We have a mock for the form with the `FormGroupDirective` using that we don't need to create a component for testing.

Reply:
That directive is in the `edit-content` library. I'll use it when we move the component to that library. For now, I'll add a comment.